client/src/Components/App.js

```diff
@@ -10,12 +7,7 @@ class App extends Component {
     return (
       <div className="App">
         <header className="App-header">
-        <HeaderComp />
```

Comment:
These removed lines might conflict with app.js changes in #49 since they are so close to each other.  It might make things go smoother if you chat with @donia98 as you both get ready to merge (you might decide to skip your app.js changes in this PR and put them in a subsequent PR, if that makes sense)